### PR TITLE
Create build_info.md

### PR DESCRIPTION
**Pull request by Winfidonarleyan**
## Related Issue
- azerothcore/azerothcore-wotlk#5626

### CODE REVIEW

File: docs/build_info.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+[<-Back-to:Auth](database-auth.md)
+
+**The \`build_info\` table**
+
+**Structure**
+
+| Field                 | Type         | Attributes | Key | Null | Default           | Extra          | Comment    |
+|-----------------------|--------------|------------|-----|------|-------------------|----------------|------------|
+| [build][1]            | INT(11)      |            | PRI | NO   |                   |                | Identifier |
```
Comment (Kitzunu) [inline docs/build_info.md:11]: INT

File: docs/build_info.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+| [build][1]            | INT          |            | PRI | NO   |                   |                | Identifier |
+| [majorVersion][2]     | INT          |            |     | YES  |                   |                |            |
+| [minorVersion][3]     | INT          |            |     | YES  |                   |                |            |
+| [bugfixVersion][4]    | INT          |            |     | YES  |                   |                |            |
+| [hotfixVersion][5]    | CHAR(3)      |            |     | YES  |                   |                |            |
+| [winAuthSeed][6]      | VARCHAR(32)  |            |     | YES  |                   |                |            |
+| [win64AuthSeed][7]    | VARCHAR(32)  |            |     | YES  |                   |                |            |
+| [mac64AuthSeed][8]    | VARCHAR(32)  |            |     | YES  |                   |                |            |
+| [winChecksumSeed][9]  | VARCHAR(40)  |            |     | YES  |                   |                |            |
+| [macChecksumSeed][10] | VARCHAR(40)  |            |     | YES  |                   |                |            |
```
Comment (Kitzunu) [inline docs/build_info.md:20]:
```suggestion
| [majorVersion][2]     | INT          |            |     | YES  | NULL              |                |            |
| [minorVersion][3]     | INT          |            |     | YES  | NULL              |                |            |
| [bugfixVersion][4]    | INT          |            |     | YES  | NULL              |                |            |
| [hotfixVersion][5]    | CHAR(3)      |            |     | YES  | NULL              |                |            |
| [winAuthSeed][6]      | VARCHAR(32)  |            |     | YES  | NULL              |                |            |
| [win64AuthSeed][7]    | VARCHAR(32)  |            |     | YES  | NULL              |                |            |
| [mac64AuthSeed][8]    | VARCHAR(32)  |            |     | YES  | NULL              |                |            |
| [winChecksumSeed][9]  | VARCHAR(40)  |            |     | YES  | NULL              |                |            |
| [macChecksumSeed][10] | VARCHAR(40)  |            |     | YES  | NULL              |                |            |
```

File: docs/build_info.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+[1]: #build
+[2]: #majorVersion
+[3]: #minorVersion
+[4]: #bugfixVersion
+[5]: #hotfixVersion
+[6]: #winAuthSeed
+[7]: #win64AuthSeed
+[8]: #mac64AuthSeed
+[9]: #winChecksumSeed
+[10]: #macChecksumSeed
```
Comment (Kitzunu) [inline docs/build_info.md:31]:
```suggestion
[2]: #majorversion
[3]: #minorversion
[4]: #bugfixversion
[5]: #hotfixversion
[6]: #winauthseed
[7]: #win64authseed
[8]: #mac64authseed
[9]: #winchecksumseed
[10]: #macchecksumseed
```

Comment (Kitzunu) [reply docs/build_info.md:31]: Only this need to be lowercase for it to work

Comment (Winfidonarleyan) [reply docs/build_info.md]: Okay, big thanks

**Review by Helias — APPROVED** [review]

**Comment by Helias** [comment]
I missclicked the approve

**Comment by Helias** [comment]
let's merge so 👍

**Comment by Kitzunu** [comment]
no not before AC PR

**Pull request merged**